src/main/kotlin/frc/robot/lib/MarkerCommands.kt

```diff
@@ -0,0 +1,11 @@
+package frc.robot.lib
+
+import edu.wpi.first.wpilibj.DataLogManager
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.Commands
+
+const val ABNORMAL_EVENT_NAME = "EVENT"
+
+fun markEvent(eventName: String): Command = Commands.runOnce({ DataLogManager.log(eventName) })
```

Comment:
Correct me if I'm wrong but it only saves the values to a log file on the rio. It should also record the output to advantage scope.